packages/docs/gatsby-node.js

```diff
@@ -1,12 +1,21 @@
 const fs = require("fs");
+const path = require("path");
 const MonacoWebpackPlugin = require("monaco-editor-webpack-plugin");
 
+const { NODE_ENV } = process.env;
+
 const REPOSITORY = "https://github.com/JSMonk/hegel";
 const STD_LIB_PATH = "@hegel/typings/standard/index.d.ts";
 const STD_LIB_CONTENT = fs
   .readFileSync(require.resolve(STD_LIB_PATH), "utf8")
   .replace(/`/g, "");
 
+const resolve = NODE_ENV !== 'production' ?  {
+  alias: {
+    '@hegel/core': path.resolve(__dirname, '../../core/build/'),
+  },
+} : {};
```

Comment:
```suggestion

const LOCAL_CORE_BUILD = path.resolve(__dirname, '../../core/build/')

const resolve = NODE_ENV !== 'production' && fs.existsSync(LOCAL_CORE_BUILD)?  {
  alias: {
    '@hegel/core': LOCAL_CORE_BUILD,
  },
} : {};
```

Reply:
great suggestions - I added it
thank you